dockerfiles/testing-environment/Laravel94/app/app/Http/Controllers/ExampleController.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Routing\Controller as BaseController;
+use Illuminate\Support\Facades\DB;
+
+class ExampleController extends BaseController
+{
+    public function example()
+    {
+        return "Hi from Laravel 5.7 app!\n";
```

Comment:
Laravel 5.7, huh? :)

Reply:
Nice find, but that was basically the source, we just copied the old version and updated it